app/code/spec/Inviqa/SymfonyContainer/Model/StoreConfigCompilerPassSpec.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace spec;
+
+use Bridge\MageApp;
+use Bridge\MageStore;
+use Inviqa_SymfonyContainer_Model_StoreConfigCompilerPass;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+class Inviqa_SymfonyContainer_Model_StoreConfigCompilerPassSpec extends ObjectBehavior
+{
+    function let(MageApp $app, MageStore $mageStore)
+    {
+        $mageStore->getConfig('store/config/key')->willReturn('some_value');
+
+        $app->getStore()->willReturn($mageStore);
+        $services = [
+            'app' => $app
+        ];
+
+        $this->beConstructedWith($services);
+    }
+
+    function it_does_not_add_an_argument_to_service_def_if_tag_does_not_exist(ContainerBuilder $container)
+    {
+        $container->findTaggedServiceIds(Inviqa_SymfonyContainer_Model_StoreConfigCompilerPass::TAG_NAME)->willReturn([]);
```

Comment:
Same comment about the alias. It would make the code neater.
```php
use Inviqa_SymfonyContainer_Model_StoreConfigCompilerPass as StoreConfigCompilerPass
```